docs/edgeql/select.rst

```diff
@@ -843,6 +843,48 @@ shapes just like a non-computed link.
   } filter .name = "Iron Man";
 
 
+Using backlinks to find items
+-----------------------------
+
+Another handy use for backlinks is using them to filter and find items
+such as during an update. This can work as a nice shortcut when you have
```

Comment:
Since we're in the `select` docs, I think our example needs to be a `select` rather than an `update`. We have already talked about computed fields at this point on the page, so your example can use one and end up being practically the same. Just `select` instead of `update` and, since you're not actually updating anything, add a shape with a computed that gives your new name to Spiderman nemeses. Something like this:

```
select Villain filter .<characters =
  <Movie><uuid>'6c60c28a-5c03-11ee-99ff-dfa425012a05' {
    name := .name ++ ', who got to see Spider-Man!'
};
```
Then you can just show the output of this and skip the `select`.

Your without backlink example could look like this:
```
with movie := 
  <Movie><uuid>'6c60c28a-5c03-11ee-99ff-dfa425012a05',
select movie.characters[is Villain] {
  name := .name ++ ', who got to see Spider-Man!'
};
```

Not tested, but I think it should work. At any rate, I think you get what I'm going for. Just adapting what you have to a `select` to better fit the context.